README.md

```diff
@@ -1,9 +1,82 @@
 # MILLION DOLLAR DREAM
 Every man has a price.
 
-This creates a bloom filter to store and lookup file hashes in a space efficent
+This project assumes you have a whole boat load (10k+?) of files you have calculated
+a baseline of and that you want to compare to see if any have changed.  This 
+creates a bloom filter to store and lookup file hashes in a space efficient
 manner. 
 
+## OPTIONS
+
+```bash
+python3 ./million_dollar_dream.py 
+usage: ./million_dollar_dream.py <calculate|lookup|fromfile|filters> <filterfile> <file1> [file2 ...]
+```
+
+* calculate - given an input of file(s), calculate all the hashes and store them in the filter file
+* lookup - tbd
+* fromfile - tbd
+* filters - given an input of file(s), see if any of them have changed based on the  the filter 
+file. Returns `is not in filter` and `is in filter` for every file input
+
+## USAGE
+
+### Calculate 
+The first step is to create your bloom filters as a baseline of what
+you'll use in the future.  You may choose to have multiple filter files
+you'll use or one master one.  Assuming you have your source files in
+`./source_dir` which you want to make hashes of to store in your filter. 
+This dir has 3 files:
+
+```bash
+./source_dir
+./source_dir/a_file.txt
+./source_dir/b_file.txt
+./source_dir/c_file.txt
+```
+
+Also, let's assume you want store your filter in `/tmp/bloom`. So if

```

Comment:
You can store filters anywhere, but the tool comes with some already in `million_dollar_dream/filters`. That is also where the `filters` command stores files, so it might be a good idea to point people there.